BiasAndFairnessModule/src/data_loader.py

```
@@ -40,7 +40,9 @@ def load_data(self) -> pd.DataFrame:
             try:
                 dataset = load_dataset(self.dataset_config.source, split=split)
             except Exception as e:
-                raise ValueError(f"Could not load split:'{split}' for dataset '{self.dataset_config.source}'. Error: {e}")
+                raise ValueError(
+                    f"Could not load split:'{split}' for dataset '{self.dataset_config.source}'. Error: {e}"
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)